hooks/terraform_wrapper_module_for_each.sh

```diff
@@ -0,0 +1,409 @@
+        echo "--module-repo-org=...         - Module repository organization (e.g., 'terraform-aws-modules'). (Optional)"
+        echo "--module-repo-shortname=...   - Short name of the repository (e.g., for 'terraform-aws-s3-bucket' it should be 's3-bucket'). (Optional)"
+        echo "--module-repo-provider=...    - Name of the repository provider (e.g., for 'terraform-aws-s3-bucket' it should be 'aws'). (Optional)"
+        echo "--dry-run                     - Whether to run in dry mode. By default, files will be overwritten."
+        echo "--verbose                     - Show verbose output."
+        echo
+        echo "Example:"
+        echo "--module-dir=modules/object   - Generate wrapper for one specific submodule."
+        echo "--module-dir=.                - Generate wrapper for the root module."
+        echo "--module-repo-org=terraform-google-modules --module-repo-shortname=network --module-repo-provider=google  - Generate wrappers for repository available by name \"terraform-google-modules/network/google\" in the Terraform registry and it includes all modules (root and in \"modules/*\")."
+        exit 1
+        ;;
+    esac
+
+  done
+
+  if [[ -z "$root_dir" ]]; then
+    echo "--root-dir can't be empty. Remove it to use default value."
+    exit 1
+  fi
+
+  if [[ -z "$wrapper_dir" ]]; then
+    echo "--wrapper-dir can't be empty. Remove it to use default value."
+    exit 1
+  fi
+
+  if [[ -z "$module_repo_org" ]]; then
+    echo "--module-repo-org can't be empty. Remove it to use default value."
+    exit 1
+  fi
+
+  if [[ -z "$module_repo_shortname" ]]; then
+    echo "--module-repo-shortname can't be empty. It should be part of full repo name (eg, s3-bucket)."
+    exit 1
+  fi
+
+  if [[ -z "$module_repo_provider" ]]; then
+    echo "--module-repo-provider can't be empty. It should be name of the provider used by the module (eg, aws)."
+    exit 1
+  fi
+
+  if [ ! -d "$root_dir" ]; then
+    echo "Root directory $root_dir does not exist!"
+    exit 1
+  fi
+
+  declare -a all_module_dirs=("./")
+  # Find all modules directories if nothing was provided via "--module-dir" argument
+  if [[ -z "$module_dir" ]]; then
+    # shellcheck disable=SC2207
+    all_module_dirs+=($(cd "${root_dir}" && find . -path '**/modules/*' -maxdepth 2 -type d -print))
+  else
+    all_module_dirs=("$module_dir")
+  fi
+
+  for module_dir in "${all_module_dirs[@]}"; do
+
+    # Remove "./" from the "./modules/iam-user" or "./"
+    module_dir="${module_dir//.\//}"
+
+    full_module_dir="${root_dir}/${module_dir}"
+    # echo "FULL=${full_module_dir}"
+
+    if [ ! -d "$full_module_dir" ]; then
+      echo "Module directory $full_module_dir does not exist!"
+      exit 1
+    fi
+
+    # Remove "modules/" from "modules/iam-user"
+    module_name="${module_dir//modules\//}"
+    if [ "$module_name" == "" ]; then
```

Comment:
align formatting
```suggestion
    if [[ "$module_name" == "" ]]; then
```
also this conditional may probably use `[[ ! $module_name ]]` (which will check for empty value along with undefined var per se.